### PR TITLE
FIXED: styling, focus container

### PR DESCRIPTION
**Pull request by Adarsh-Patel-Dev**
image
image

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -176,7 +177,7 @@ input:focus{
   font-weight: 200;
 }
 
-.focuscontainer{
+.focus-container{
```
Comment (theVivek-kumar) [inline src/App.css:180]: you  can do is just a little bit of formatting

File: src/pages/homePage/HomePage.jsx [hunk 2]
```diff
@@ -9,16 +10,18 @@ function HomePage() {
     const [ display, setDisplay ] = useState("block")
```
Comment (theVivek-kumar) [inline src/pages/homePage/HomePage.jsx:9]: use good name for setState

File: src/pages/mainPage/MainPage.jsx [hunk 3]
```diff
@@ -1,17 +1,36 @@
...
+      setCheck(true)
+    }
+
+  }
     const myfocus = localStorage.getItem('focus');
   return (
-    <div>
+    <div className='main-container main'>
         <Clock/>
-        <h1>Good Evening, {localStorage.getItem('userName')}.</h1>
```
Comment (theVivek-kumar) [inline src/pages/mainPage/MainPage.jsx:10]: gave some good function name

**Review by theVivek-kumar — APPROVED** [review]
can be merged  not a major change required

File: src/pages/mainPage/MainPage.jsx [hunk 4]
```diff
@@ -1,17 +1,36 @@
...
 
 function MainPage() {
+  const [ style, setStyle ] = useState('none')
+  const [ check, setCheck ] = useState(true)
+
+  function focusHandler(e){
+    if(check){
+      setCheck(false)
+      setStyle('line-through')
+      console.log(check)
```
Comment (JuvitaSaini09) [inline src/pages/mainPage/MainPage.jsx]: no need to keep console.log

**Review by JuvitaSaini09 — APPROVED** [review]
looks good to merge

**Review by muskaanshah — APPROVED** [review]
Small PR, clean code, good to merge

File: src/components/quoteComponent/quote.css [hunk 5]
```diff
@@ -4,8 +4,9 @@
 }
 
 .quote-container{
-    position: absolute;
-    bottom: 0;
-    margin: auto;
-    width: 100%;
+    /* display: flex;
```
Comment (arjun-computer-geek) [inline src/components/quoteComponent/quote.css]: remove unused code it will good

File: src/pages/homePage/HomePage.jsx [hunk 6]
```diff
@@ -1,6 +1,7 @@
 
 import { React, useState} from 'react'
 import { Clock, Quote} from '../../components'
+import { useNavigate } from 'react-router-dom'
 // import Quote from '../../components/quoteComponent/Quote'
```
Comment (arjun-computer-geek) [inline src/pages/homePage/HomePage.jsx]: like these

**Review by arjun-computer-geek — APPROVED** [review]
All are good. Good code of conduct.

**Comment by Adarsh-Patel-Dev** [comment]
completed P0 features

**Comment by Adarsh-Patel-Dev** [comment]
completed P0 features

**Pull request merged**